src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs

```diff
@@ -49,6 +49,7 @@ public FhirCosmosResourceWrapper(ResourceWrapper resource)
             IReadOnlyCollection<KeyValuePair<string, string>> lastModifiedClaims,
             string searchParameterHash = null)
             : base(resourceId, versionId, resourceTypeName, rawResource, request, lastModified, deleted, searchIndices, compartmentIndices, lastModifiedClaims, searchParameterHash)
+            ////: base(resourceId, versionId, ResourceKey.NameToId(resourceTypeName), rawResource, request, lastModified, deleted, searchIndices, compartmentIndices, lastModifiedClaims, searchParameterHash)
```

Comment:
Is it intentional to leave this here commented out? Same goes for the using statements and other lines in this file.